test/mzworkflows_examples/mzworkflows.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import os
+from unittest.mock import patch
+
+from materialize.mzcompose import (
+    Kafka,
+    Materialized,
+    SchemaRegistry,
+    Testdrive,
+    Workflow,
+    Zookeeper,
+)
+
+confluents = [Zookeeper(), Kafka(), SchemaRegistry()]
+
+versioned_mz = Materialized(image="materialize/materialized:${MZ_VERSION}")
+mz_instances = list(
+    map(
+        lambda i: Materialized(
+            name=f"materialized{i}",
+            data_directory=f"/share/materialized{i}",
+            port=6875 + i,
+        ),
+        [1, 2],
+    )
+)
```

Comment:
I would recommend using a list comprehension here, which is generally preferred in Python over `map`:

```
mz_instances = [
    Materialized(name=f"materialized{i}", ...)
    for i in [1, 2]
]
```

Reply:
Thanks, fixed.